humans/static/javascripts/update_user_form.js

```diff
@@ -36,14 +36,25 @@ $(document).ready(function(){
     if ($('.radio_button:checked').val() == "keyserver") {
       $("#keyserver_url").show();
       $("#public_key").hide();
-      $("#public_key").val('')
     } else {
       $("#keyserver_url").hide();
       $("#public_key").show();
-      $("#keyserver_url").val('')
     }
   });
-
+  if($("#id_keyserver_url").val().length != 0) {
+    $("#keyserver_option").prop("checked", true);
+    $("#keyserver_url").show();
+    $("#public_key").hide();
+  }
+  $("#form").submit(function(event){
+    if ($('.radio_button:checked').val() == "keyserver") {
+      $("#id_public_key").val('');
+      $("#id_public_key").text('');
```

Comment:
This is actually strange but I noticed that doing the .val('') wasn't enough and that it still saved the text that was in the keys. Applying the .text('') was the only way that make everything work and that actually erased the previous text.